apps/wing-console/console/ui/package.json

```diff
@@ -37,6 +37,7 @@
     "@trpc/react-query": "^10.30.0",
     "@trpc/server": "^10.30.0",
     "@wingconsole/design-system": "workspace:^",
+    "@wingconsole/server": "workspace:^",
```

Comment:
I think it shouldn't be necessary to have `@wingconsole/server` as a dep instead of devDep anymore, since we only use types and they have been corrected.

Reply:
done!